apps/web/src/common/export.ts

```diff
@@ -33,7 +33,28 @@ export async function exportToPDF(
 
   return new Promise<boolean>((resolve) => {
     const iframe = document.createElement("iframe");
-    iframe.srcdoc = content.replaceAll(/<p(.+?)><\/p>/gm, "<p$1><br/></p>");
+
+    const replaceableAttributes = {
+      'data-align="left"':
+        'style="margin-right:auto;margin-left:0;display: block;"',
+      'data-align="right"':
+        'style="margin-left:auto;margin-right:0;display: block;"',
+      'data-align="center"':
+        'style="margin-left:auto;margin-right:auto;display: block;"',
+      'data-float="true" data-align="right"': 'align="right"',
+      'data-float="true" data-align="left"': 'align="left"'
+    };
+
+    let html = content.replaceAll(/<p(.+?)><\/p>/gm, "<p$1><br/></p>");
+
+    for (const attribute of Object.keys(replaceableAttributes)) {
```

Comment:
```suggestion
    for (const attribute in replaceableAttributes) {
```
No need for `Object.keys`.